development/print_db_data.py

```diff
@@ -122,50 +122,36 @@ def print_production_lines():
 def print_verbose_lines():
 	print('Data has been moved, this view is unavailable for now')
 	return
-	def _output_helper_prodlines(string, list):
```

Comment:
People who would like to obtain information about UH's data files in a condensed, easy-to-understand format. That should include all developers at the very least.